### ISSUE TITLE
Square has Bad Equation

### ISSUE THREAD

**Issue by bhsmither** [cubecart/v6#2650]
In Square 1.1.5, an edit was made to cast the result of a function to an integer:
```
(int)$this->_basket['total']*100
```
This is not correct. (At least for PHP 7.4.) The basket total is getting cast before getting multiplied.

The correct statement is:
```
(int)($this->_basket['total']*100)
```

**Comment by bhsmither** [comment]
Even so, there may still be an issue with certain versions of PHP 7 that specifically affects values ending in `.99`.

See the comments at: https://www.php.net/manual/en/function.intval

**Comment by abrookbanks** [comment]
This was fixed with 1.1.6. :o)
